wordpress-mobile/WordPress-Android
# Jetpack focus: remove tab bar and dashboard

Part of  #17339

## Descriptions
This PR partially removes the jetpack features for users in P4. It removes the tab bar and the visible entry points for Jetpack features. This PR also disables Quick Start and the onboarding prompt in the UI in Phase 4. 

## What are the changes 
- [x] Remove the tab bar, dashboard, and initial screen setting in Jetpack feature removal phase 4 
    - [x]  Remove features
      - [x]  Stats
      - [x]  Reader
    - [x] Remove Menu Items 
      - [x]  Activity Log
      - [x]  Jetpack Social
      - [x]  Backup
      - [x]  Scan
      - [x] Themes 
      - [x] People
     - [x]  Disable Quick Start


## UI 
|  Landing page in Normal Phase | Landing page in Jetpack feature removal phase 4 | 
|---|---|
| ![landing_page_light](https://user-images.githubusercontent.com/17463767/212674510-15e415ca-5ba5-4f40-8ce4-4a9abcfd6a38.png)|![landing_page_phase4_light](https://user-images.githubusercontent.com/17463767/212674519-b34e3701-1945-4d49-84e5-dee820593c19.png)| 


### Testing Instructions
- Run the app
- Open the debug menu and disable all the Jetpack Features Removal flags
- Kill the app
- Relaunch the app
- Make sure the app is behaving normally including quick start 
- Open the debug menu and enable "jp_removal_four"
- Kill the app
- Relaunch the app
- Verify that the tap bar is gone and all Jetpack features are removed.


## Regression Notes
1. Potential unintended areas of impact
App not behaving properly 

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing 

3. What automated tests I added (or what prevented me from doing so)


PR submission checklist:
